apertium-init.py

```diff
@@ -26,250 +26,4 @@
WuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^ClWuATOMC^Cl
```

Comment:
It broke because it looks like you deleted all of this.

Reply:
I didn't remove anything manually, I just ran `make` and committed the changes.
Should I run `make` again, or revert `apertium-init.py` to its previous state (before I ran `make`)?

Reply:
Revert and re-run `make`. Maybe you're using the wrong Python version or something?

Reply:
I that tried, but the build still fails. I have Python 2.7 and 3.6 on my machine. Do you want to try and run `make` from your end and push the updated `apertium-init.py`?

Reply:
Alas, we'll do that.